document_stores/elasticsearch/tests/test_document_store.py

```diff
@@ -1,6 +1,10 @@
 # SPDX-FileCopyrightText: 2023-present Silvano Cerza <silvanocerza@gmail.com>
 #
 # SPDX-License-Identifier: Apache-2.0
+
+# ruff: noqa: S311
```

Comment:
Can we move this ignore to the line that actually uses `random`?

Reply:
I had some trouble doing it, but now I have managed.